modules/microshift-low-latency-rhel-edge-blueprint-rtk.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="microshift-low-latency-blueprint-rhel-edge-rtk_{context}"]
+= Reference blueprint for installing {op-system-rt-kernel} in a {op-system-ostree} image
+
+An image blueprint is a persistent definition of the required image customizations that enable you to create multiple builds. Instead of reconfiguring the blueprint for each image build, you can edit, rebuild, delete, and save the blueprint so that you can keep rebuilding images from it.
+
+* A blueprint groups the images that were built from it. This means that you can have easier access to related images, especially when you are dealing with lots of images.
+
+* Customizations stored in a blueprint are always applied across all builds resulting from the blueprint. This is true even if the resulting builds, and images from those builds, have different content versions.
+
+* You can create a blueprint with a name, specify the packages that you want to install, and define other customizations, such as including {microshift-short} and low-latency settings.
+
+* Images created from a blueprint take the name of the blueprint unless you define a name.
```

Comment:
Do we need this information here? Wouldn't the section about creating the blueprints for R4E be a better place and only have the example here?

Reply:
This is from the rtk docs for RHEL, not Edge, so I didn't know if it applied to all blueprints. Knowing what a blueprint is seems like reference info to me, but I can look around and see if we have a better spot.

Reply:
Pulled out everything but the minimum required by mod docs. Deal with it another time and place.